pkg/bindings/test/images_test.go

```diff
@@ -346,7 +346,7 @@ var _ = Describe("Podman images", func() {
 		}
 
 		//	Search with a fqdn
-		reports, err = images.Search(bt.conn, "quay.io/libpod/alpine_nginx", nil)
+		reports, err = images.Search(bt.conn, "quay.io/podman/stable", nil)
```

Comment:
Nothing at all to change now, but we might want to hit up quay.io/podman/hello for tests like this one and below.  A smaller image by far.  Something to consider for the future.